src/renderer/components/layout/sidebar.tsx

```diff
@@ -44,29 +45,50 @@ export class Sidebar extends React.Component<Props> {
     crdStore.reloadAll();
   }
 
-  renderCustomResources() {
-    if (crdStore.isLoading) {
+  @computed get crdSubMenu(): React.ReactNode {
+    if (!crdStore.isLoaded) {
```

Comment:
I don't think that we should display the spinner if loading has failed.

Reply:
I'd prefer to keep it as is since it fixes a bug (see video1) with reloading CRD definitions each time (otherwise more files needs to be tweaked)

Reply:
Where in the video?

Reply:
When I'm clicking second time at `Definitions` you can see loading indicator and jumping scroll for a short time.